imports/api/eligibleClients/methods.js

```diff
@@ -23,6 +23,21 @@ Meteor.methods({
     return hc.api('house-matching-v2').getEligibleClientsPage(pageNumber, pageSize, sort, order);
   },
 
+  updateBonusScore(clientId, bonusScore) {
+    logger.info(`METHOD[${this.userId}]: updateScores(${Array.isArray(clientId)}, ${clientId})`); // eslint-disable-line max-len
+    check(clientId, Match.OneOf(String, [String])); // eslint-disable-line new-cap
+    if (!Roles.userIsInRole(this.userId, ClientsAccessRoles)) {
+      throw new Meteor.Error(403, 'Forbidden');
+    }
+
+    const hc = HmisClient.create(this.userId);
+    // get client details
+    const eligibleClient = hc.api('house-matching').getEligibleClient(clientId);
```

Comment:
what if clientId is an array? shouldnt we have clientId.forEach(...) is this case?